Common/tests/TestBase/TaskWatcherTestBase.cs

```diff
@@ -473,4 +473,50 @@ await foreach (var cur in watchEnumerator.WithCancellation(cts.Token)
                       newResults[2]);
     }
   }
+
+  [Test]
+  public async Task WatchTaskStatusUpdateWithComplexFilterShouldSucceed()
+  {
+    if (RunTests)
+    {
```

Comment:
```suggestion
    if (!RunTests)
    {
      return;
    }
```

Reply:
As it is done like that in every test, it is not the place to change that.